lib/topological_inventory/ansible_tower/operations/core/service_order_mixin.rb

```diff
@@ -11,7 +11,7 @@ def order
             task_id, service_offering_id, service_plan_id, order_params = params.values_at(
               "task_id", "service_offering_id", "service_plan_id", "order_params")
 
-            update_task(task_id, :state => "running", :status => "ok", :context => {})
+            update_task(task_id, :state => "running", :status => "ok", :context => {:info => 'Task started'})
```

Comment:
@slemrmartin is context.info something that will actually be used or is this just to fix the error?
Could we just not pass any `:context` if it isn't going to contain a payload?